lib/utils/manage_updates.dart

```diff
@@ -0,0 +1,102 @@
+import 'dart:io';
+
+import 'package:sidekick/utils/open_link.dart';
+import 'package:archive/archive.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:github/github.dart';
+//import 'package:hive/hive.dart';
+import 'package:http/http.dart' as http;
+import 'package:oktoast/oktoast.dart';
+import 'package:path_provider/path_provider.dart' as provider;
+import 'package:version/version.dart';
+
+import '../components/molecules/update_card.dart';
+import '../version.dart';
+
+void downloadRelease(String release) async {
+  var directory = await provider.getDownloadsDirectory();
+  var fileLocation =
+      "${directory.absolute.path}${Platform.pathSeparator}CRel-$release.";
+  var url =
+      "https://github.com/aguilaair/Companion/releases/download/$release/${Platform.operatingSystem.toLowerCase()}-$release.zip";
```

Comment:
Need to update this for the URL on this repo. Also can we move this into the constants file?

Reply:
Done!